plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -195,6 +195,22 @@ static inline bool HasOptimizedBRCSupport(const mfxPlatform &platform,
 	UNUSED_PARAMETER(rateControl);
 	return false;
 }
+
+static inline bool HasAV1ScreenContentSupport(const mfxPlatform &platform,
+					      const mfxVersion &version)
+{
+#if (MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 12) || MFX_VERSION_MAJOR > 2
+	// AV1_SCREEN_CONTENT_TOOLS API is introduced in VPL version 2.11
+	if ((version.Major >= 2 && version.Minor >= 12) || version.Major > 2)
```

Comment:
If this is introduced in 2.11, then shouldn't the version checks check for >= 2.11 instead of 2.12?

Reply:
The checks for the AV1_SCEEN_CONTENT_TOOLS are all in version 2.11, but the checks for the platforms (Lunarlake, Alderlake, and Arrowlake) are in 2.12

Reply:
So `ExtAV1ScreenContentTools` is available in VPL 2.11, but the platform enums are only defined in 2.12 or the runtime requirement is 2.12+?

Reply:
yes, the platform enums are only defined in 2.12, so the runtime requirement is also 2.12

Reply:
The comment may need to be updated to make that clearer.

Reply:
I moved this specific comment to the AV1 SCC code in the InitParams function and changed the comment here to be more clear.